bedrock/firefox/templates/firefox/hello/index-b.html

```diff
@@ -2,7 +2,7 @@
  # License, v. 2.0. If a copy of the MPL was not distributed with this
  # file, You can obtain one at http://mozilla.org/MPL/2.0/. #}
 
-{% extends "firefox/hello/index-2016.html" %}
+{% extends "firefox/hello/index.html" %}
 
 {% set_lang_files "firefox/hello/index-2016" %}
```

Comment:
Changing the name of the lang file is more trouble than it's worth. The directory is still the same, so this should be uneffected.